src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -120,12 +120,24 @@ def get_pool_list_with_stats(cls, application=None):
 
     @classmethod
     def get_pool_name_from_id(cls, pool_id):
+        pool = cls.get_pool_by_attribute('pool', pool_id)
```

Comment:
Please add typo infos, too. This will help devs to understand what methods require as parameters and what is returned.